addons/mail/models/discuss/discuss_channel.py

```diff
@@ -273,19 +273,32 @@ def write(self, vals):
             failing_channels = self.sudo().filtered(lambda channel: channel.channel_type != vals.get('channel_type'))
             if failing_channels:
                 raise UserError(_('Cannot change the channel type of: %(channel_names)s', channel_names=', '.join(failing_channels.mapped('name'))))
+        notifications = []
+        for channel in self:
+            current_val = channel.read(vals.keys())[0]
+            diff = {}
+            for key in vals.keys():
+                if current_val.get(key) != vals.get(key) and key != "image_128":
+                    diff[key] = vals[key]
+                    notifications.append([channel, "mail.record/insert", {
+                        "Thread": {
+                            "id": current_val["id"],
+                            "model": "discuss.channel",
+                            **diff
+                        }
+                    }])
```

Comment:
Slightly out-of-scope of PR, but I think `image_128` could be grouped in same notification

Reply:
Yes and no, as I said, you don't want the image_128 field in the bus notification as base64. You want to fill the `avatarCacheKey`.
You could try to group the notification, but I m not sure the code would be better. Having a dedicated logic for `image_128` isn't that bad.